mica/archive/aca_l0.py

```diff
@@ -95,8 +114,21 @@ def get_options():
     return opt
 
 
+def _mask(n):
```

Comment:
This doesn't appear in this file.  Orphaned?

Reply:
yes, I see it is not used. I must have used it at the beginning. In the end, this mask does not enter in the magnitude calculation, so it is not used anywhere. I will remove it.